provisioning/dashboards/ea-advanced-dashboard.json

```diff
@@ -2043,7 +2043,7 @@
           {
             "matcher": {
               "id": "byName",
-              "options": "seeking"
+              "options": "  seeking"
```

Comment:
is this leading blank space ok?

Reply:
Yes, check the previous fields 

https://github.com/System73/edge-analytics-grafana-setup/pull/14/files#diff-98d1c79b91614e628ea937e6fe2164dd919890ebe898e142e544d406d1856254L1674

The lack of it is caused by adding that matcher manually on the JSON, it doesn't cause issues but lets keep consistency with how Grafana creates them and exports them.